README.md

```diff
@@ -8,21 +8,27 @@ Several of our examples are inspired by the [Scanpy tutorials](https://scanpy.re
 
 Dataset sizes for single-cell genomics studies are increasing, presently reaching millions of cells. With RAPIDS, it becomes easy to analyze large datasets interactively and in real time, enabling faster scientific discoveries.
 
-## Installation 
+## Installation
 
 ### Docker container
 A container with all dependencies, notebooks and source code is available at https://hub.docker.com/r/claraparabricks/single-cell-examples_rapids_cuda11.0.
 
 Please execute the following commands to start the notebook and follow the URL in the log to open Jupyter web application.
 
+```bash
```

Comment:
I suggest removing this since the use of the launch script is described separately.